src/averageAges.js

```diff
@@ -36,8 +47,19 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+
+function calculateWomenAverageAge(people, withChildren = false) {
+  const women = people.filter(person => {
+    return withChildren
+      ? people.find(child => person.name === child.mother && person.sex === 'f')
+      : person.sex === 'f';
+  });
+
+  const sumOfAges = women
```

Comment:
this calculation repeated in several function so you can create separate function and reuse it